ads/_config.js

```diff
@@ -193,6 +193,12 @@ const adConfig = jsonConfiguration({
     preconnect: 'https://ad.ad-stir.com',
   },
 
+  'adstyle': {
+    renderStartImplemented: true,
```

Comment:
Great that you decided to implement this, but I do not see this callback being executed. Can you confirm you are calling this in you external JS? If you are unable to do so you should remove this flag (keep the prefetch and preconnect) as you will suffer a small penalty if the runtime is waiting for this signal but it is never called.